DataImport.Models.Tests/appsettings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ConnectionStrings": {
-    "defaultConnection": "Data Source=(local);Initial Catalog=EdFi_DataImport_Test;Trusted_Connection=True"
-  }
-}
\ No newline at end of file
+    "ConnectionStrings": {
```

Comment:
Since this is a test project, it is probably okay to keep the `TrustServerCertificate=True;`